scripts/zones/Lower_Jeuno/npcs/Treasure_Coffer.lua

```diff
@@ -532,6 +1619,7 @@ local argumentKeyItems =
 }
 
 entity.onTrigger = function(player, npc)
+    local eligableNexusCape = (xi.settings.main.ENABLE_ACP * xi.settings.main.ENABLE_AMK * xi.settings.main.ENABLE_ASA) ~= 0
```

Comment:
Even though the values for each of these settings is 1 or 0, they're mentally treated like boolean values. I think it would be more appropriate to treat these as bool values rather than integer values that we're trying to multiply out in case one of them is zero.
```lua
local eligableNexusCape = setting1 == 1 and 
                          ... and 
                          ...

Reply:
hmm okie dokie